simply/actor.py

```diff
@@ -138,12 +139,25 @@ def create_prediction(self):
             self.pred["schedule"] = self.pred["pv"] - self.pred["load"]
 
     def update(self):
+        """Changes actor attributes according to the events in the current time step
+
+        The events can be the impact of schedule and trading on the battery soc, the bank / cost for
+        the actor in this time step, and an updated prediction which moves one time step ahead."""
         if self.battery and not self.pred.empty:
             self.get_energy()
             self.socs.append(self.battery.soc)
         self.create_prediction()
 
     def get_energy(self, _cache=dict()):
+        """Charge or discharge the battery depending on the current planned schedule.
+
+        This function needs to be called once per time step to track the energy inside of the
+        battery. It takes the planned, i.e. predicted, schedule and changes the batter soc
+        accordingly. A positive schedule charges the battery and a negative schedule discharges the
+        battery. At a later stage this function will add the traded energy amount as well.
```

Comment:
typo, clarify and shorten:
Update the battery state depending on the currently scheduled planning time step.

This function needs to be called once per time step to track the energy inside of the
battery. It takes the planned, i.e. predicted, schedule and changes the battery's SOC
accordingly. 

-  (already described in battery.charge) ~~A positive schedule charges the battery and a negative schedule discharges the battery.~~
-  (unclear, unnecessary?)~~At a later stage this function will add the traded energy amount as well.~~